deploy/helm-preview/services/workflows-service/templates/deployment.yaml

```
@@ -0,0 +1,74 @@
+              command: ['npm','run','db:init']
+              {{- with .Values.workflowService.envFrom }}
+              envFrom:
+              {{- toYaml . | nindent 14 }}
+              {{- end }}
+            {{- end }}
+            {{- if .Values.workflowService.prismaMigrate.enabled }}
+            initContainers:
+              - name: wait-for-migration
+                image: pegasystems/k8s-wait-for:latest
+                imagePullPolicy: IfNotPresent
+                args: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)